DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -852,6 +852,25 @@ function PreFirstEntryStartMapFixes(#var(PlayerPawn) player, FlagBase flagbase,
 function PostFirstEntryStartMapFixes(#var(PlayerPawn) player, FlagBase flagbase, int start_flag)
 {
     local DeusExGoal goal;
+    local WaltonWareCrate waltonCrate;
```

Comment:
This logic for actually spawning and populating the crate should probably be in it's own function, and maybe just called alongside the call to PostFirstEntryStartMapFixes?  Feels like it might be a bit more clean and self-contained?